ui/app/templates/vault/cluster/access/methods.hbs

```diff
@@ -46,7 +46,7 @@
   </ToolbarActions>
 </Toolbar>
 
-{{#each (sort-by "path" this.authMethodList) as |method|}}
```

Comment:
<img width="1189" alt="Screenshot 2024-03-27 at 12 33 27 PM" src="https://github.com/hashicorp/vault/assets/68122737/2b7de26a-f9ec-4f35-bf22-ffaffed9aa6b">

Reply:
`soryBy` [docs](https://github.com/emberjs/ember.js/blob/v4.12.0/packages/%40ember/array/index.ts#L1425-L1442) uses `toArray()` so I replaced both of our uses with the `sortObjects` util